apps/meeting/src/containers/MeetingEventObserver/index.tsx

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import { useMeetingEvent } from 'amazon-chime-sdk-component-library-react';
+
+const MeetingEventObserver = () => {
+  const meetingEvent = useMeetingEvent();
+  console.log('Received meeting event in MeetingEventObserver', meetingEvent);
+  return null;
+}
+
+export default MeetingEventObserver;
```

Comment:
[nit] new line at the end of file

Reply:
Yup updated, I have to get some linter installed locally I think.